docs/pages/deploying-vclusters/high-availability.mdx

```diff
@@ -2,13 +2,169 @@
 title: High Availability
 sidebar_label: High Availability
 ---
+By default, vCluster runs one instance of each of its components. That’s fine for many use cases, like ones that are very ephemeral (dev environments, CI/CD, etc.). But suppose your situation requires virtual clusters with more redundancy. In that case, you can use vCluster’s High Availability feature to run multiple copies of the vCluster components so that the cluster is more resistant to individual pods failing.
 
+# Prerequisites
+For this tutorial, you will need a multi-node Kubernetes cluster. This can be a local cluster, created with tools like Minikube or kind, or a remote cluster managed or hosted by a cloud provider. Or even a bare metal cluster if you are so inclined. You just need more than one node in the cluster.
 
-vCluster supports high-availability with k3s as well as the vanilla k8s distribution. k0s is currently not supported for high availability setup of vCluster.
+You also need your kube config pointing to that cluster (you can connect and run kubectl commands against it).
 
-## k3s
+# 1. Install the vCluster client
+If you’re on a Mac and using Homebrew, you can install the vCluster CLI with this command:
 
-### Enabling High Availability
+```
+brew install vcluster
+```
+For other platforms, see the installation instructions.
```

Comment:
I think you can just put `<InstallCLIFragment/>` here and it'll show that fragment

Reply:
I wouldn't overload this page with the installation fragment as it s already long enough. 
Will just link the relative path of that docs page instead if it s fine with you